src/uu/hashsum/src/hashsum.rs

```diff
@@ -38,173 +37,16 @@ struct Options {
     algoname: &'static str,
     digest: Box<dyn Digest + 'static>,
     binary: bool,
-    check: bool,
+    //check: bool,
     tag: bool,
     nonames: bool,
-    status: bool,
-    quiet: bool,
-    strict: bool,
-    warn: bool,
+    //status: bool,
+    //quiet: bool,
+    //strict: bool,
+    //warn: bool,
```

Comment:
What's the reason for commenting them out instead of removing?

Reply:
because i will implement them next